webrtc.html

```diff
@@ -11087,16 +11087,10 @@ <h3>Introduction</h3>
         even past the associated peer connection being <a data-lt="close"
           href="#dom-rtcdatachannel-close"><code>close</code></a>d.
       </p>
-      <p>Only a few monitored objects have shorter lifetimes. For these objects,
-        their lifetime ends when they are <a href="#dfn-delete-stats">deleted</a>
-        by algorithms. At the time of deletion, a record of their statistics is
-        emitted in a single <code><a>statsended</a></code> event containing an
-        <code><a>RTCStatsReport</a></code> object with statistics from all
-        objects deleted at the same time. Statistics from these objects are no
-        longer available in subsequent getStats() results. The object descriptions in
-        [[WEBRTC-STATS]] describe when these monitored objects are
-        <a href="#dfn-delete-stats">deleted</a>.
-      </p>
+      <p>Only a few monitored objects have shorter lifetimes. The object
+      descriptions in [[WEBRTC-STATS]] describe when these monitored objects are
```

Comment:
Can we have a more precise reference, maybe #lifetime-considerations-for-monitored-objects?
Also, this paragraph and the previous one seem now a bit redundant, maybe we should merge them?